public/angular/translations/locale-debug.json

```diff
@@ -0,0 +1,353 @@
+  "trends_modal_help": "TRENDS_MODAL_HELP",
+  "Choose media": "CHOOSE MEDIA",
+  "Choose source": "CHOOSE SOURCE",
+  "Choose incident": "CHOOSE INCIDENT",
+  "Close": "CLOSE",
+  "Choose source media": "CHOOSE SOURCE MEDIA",
+  "Media is required.": "MEDIA IS REQUIRED.",
+  "Only one Twitter source is allowed. Instead of adding another one, you should just add keywords to the first.": "ONLY ONE TWITTER SOURCE IS ALLOWED. INSTEAD OF ADDING ANOTHER ONE, YOU SHOULD JUST ADD KEYWORDS TO THE FIRST.",
+  "Nickname": "NICKNAME",
+  "A short name for this source.": "A SHORT NAME FOR THIS SOURCE.",
+  "Nickname is required.": "NICKNAME IS REQUIRED.",
+  "Keywords": "KEYWORDS",
+  "sources_modal": {
+    "help": "sources_modal.help",
+    "rss_url": "sources_modal.rss_url",
+    "facebook_url": "sources_modal.facebook_url",
+    "elmo_url": "sources_modal.elmo_url"
+  },
+  "URL is required.": "URL IS REQUIRED.",
+  "URL is invalid.": "URL IS INVALID.",
+  "Create": "CREATE",
+  "There are no events to display.": "THERE ARE NO EVENTS TO DISPLAY.",
+  "Level": "LEVEL",
+  "Message": "MESSAGE",
+  "Create User": "CREATE USER",
+  "Sorry, but we couldn't find any users.": "SORRY, BUT WE COULDN'T FIND ANY USERS.",
+  "Role": "ROLE",
+  "Are you sure you want to delete this user?": "ARE YOU SURE YOU WANT TO DELETE THIS USER?",
+  "Are you sure you want to delete this source?": "ARE YOU SURE YOU WANT TO DELETE THIS SOURCE?",
+  "Are you sure you want to delete this trend? (All historical data will be lost)": "ARE YOU SURE YOU WANT TO DELETE THIS TREND? (ALL HISTORICAL DATA WILL BE LOST)",
+  "Recent Events": "RECENT EVENTS",
+  "Enter keywords": "ENTER KEYWORDS",
+  "Enter a nickname": "ENTER A NICKNAME",
+  "Enter url": "ENTER URL",
+  "Enter author": "ENTER AUTHOR",
+  "Enter location": "ENTER LOCATION",
+  "Enter title": "ENTER TITLE",
+  "Start date / time": "START DATE / TIME",
+  "End date / time": "END DATE / TIME",
+  "Create Source": "CREATE SOURCE",
+  "User": "USER",
+  "Profile": "PROFILE",
+  "Username is required.": "USERNAME IS REQUIRED.",
+  "Username should be less than 15 characters and should contain letters, numbers, and underscores only.": "USERNAME SHOULD BE LESS THAN 15 CHARACTERS AND SHOULD CONTAIN LETTERS, NUMBERS, AND UNDERSCORES ONLY.",
+  "Email is invalid.": "EMAIL IS INVALID.",
+  "Email is required.": "EMAIL IS REQUIRED.",
+  "Role is required.": "ROLE IS REQUIRED.",
+  "Change password": "CHANGE PASSWORD",
+  "password_min_length": "PASSWORD_MIN_LENGTH [{{passwordMinLength}}]",
+  "Passwords don't match.": "PASSWORDS DON'T MATCH.",
+  "Read": "READ",
+  "Unread": "UNREAD",
+  "Flagged": "FLAGGED",
+  "Unflagged": "UNFLAGGED",
+  "Read & Unflagged": "READ & UNFLAGGED",
+  "Total Reports": "TOTAL REPORTS",
+  "Reports/min": "REPORTS/MIN",
+  "Escalated Incidents": "ESCALATED INCIDENTS",
+  "Mark all Unread": "MARK ALL UNREAD"
+}
```

Comment:
Perhaps add this to .gitignore and put steps to generate (gulp script) and use in Readme file.